From a3aab2b7e5de39d8bfcc7422444aaafa86ce7928 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Wed, 21 Aug 2024 21:17:09 +0530
Subject: [PATCH] Update the API and target level for A15

This patch will update the target level and api
level for a15 branch to 35 and 202404 respectively

Tests Done: Build, boot and verify FIRST_API_LEVEL

Tracked-On: OAM-123279
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 groups/device-specific/caas/caas.mk                  | 2 +-
 groups/device-specific/caas/compatibility_matrix.xml | 4 ++--
 groups/device-specific/caas/framework_manifest.xml   | 6 +++---
 groups/device-specific/caas/manifest.xml             | 8 ++++----
 groups/tee/false/product.mk                          | 2 +-
 5 files changed, 11 insertions(+), 11 deletions(-)

diff --git a/groups/device-specific/caas/caas.mk b/groups/device-specific/caas/caas.mk
index 1963a62..c0c751f 100644
--- a/groups/device-specific/caas/caas.mk
+++ b/groups/device-specific/caas/caas.mk
@@ -72,7 +72,7 @@ else
 PRODUCT_SET_DEBUGFS_RESTRICTIONS := true
 endif
 {{^ota-update}}
-PRODUCT_SHIPPING_API_LEVEL := 34
+PRODUCT_SHIPPING_API_LEVEL := 35
 PRODUCT_OTA_ENFORCE_VINTF_KERNEL_REQUIREMENTS := true
 {{/ota-update}}
 {{#ota-update}}
diff --git a/groups/device-specific/caas/compatibility_matrix.xml b/groups/device-specific/caas/compatibility_matrix.xml
index 8f935e8..c09add5 100644
--- a/groups/device-specific/caas/compatibility_matrix.xml
+++ b/groups/device-specific/caas/compatibility_matrix.xml
@@ -47,10 +47,10 @@
             <instance>default</instance>
         </interface>
     </hal-->
-    <!--system-sdk>
+    <system-sdk>
         <version>VanillaIceCream</version>
     </system-sdk>
     <vendor-ndk>
         <version>35</version>
-    </vendor-ndk-->
+    </vendor-ndk>
 </compatibility-matrix>
diff --git a/groups/device-specific/caas/framework_manifest.xml b/groups/device-specific/caas/framework_manifest.xml
index de47548..ae499f3 100644
--- a/groups/device-specific/caas/framework_manifest.xml
+++ b/groups/device-specific/caas/framework_manifest.xml
@@ -139,7 +139,7 @@
         <name>android.hardware.camera.provider</name>
 	<fqname>ICameraProvider/external/0</fqname>
     </hal>
-    <!--vendor-ndk>
-        <version>34</version>
-    </vendor-ndk-->
+    <vendor-ndk>
+        <version>35</version>
+    </vendor-ndk>
 </manifest>
diff --git a/groups/device-specific/caas/manifest.xml b/groups/device-specific/caas/manifest.xml
index ff961a0..212b8ba 100644
--- a/groups/device-specific/caas/manifest.xml
+++ b/groups/device-specific/caas/manifest.xml
@@ -1,5 +1,5 @@
 {{^ota-update}}
-<manifest version="1.0" type="device" target-level="8">
+<manifest version="1.0" type="device" target-level="202404">
 {{/ota-update}}
 {{#ota-update}}
 <manifest version="1.0" type="device">
@@ -36,7 +36,7 @@
 	<fqname>ICameraProvider/external/0</fqname>
     </hal>
     <!--<kernel target-level="5"/>-->
-    <!--sepolicy>
-        <version>34.0</version>
-    </sepolicy-->
+    <sepolicy>
+        <version>202504</version>
+    </sepolicy>
 </manifest>
diff --git a/groups/tee/false/product.mk b/groups/tee/false/product.mk
index 33630cc..6dac8ec 100644
--- a/groups/tee/false/product.mk
+++ b/groups/tee/false/product.mk
@@ -1,6 +1,6 @@
 PRODUCT_PACKAGES += \
-    com.android.hardware.gatekeeper.nonsecure \
     android.hardware.security.keymint-service
+    #com.android.hardware.gatekeeper.nonsecure \
 
 #PRODUCT_PACKAGES += \
 #    android.hardware.keymaster@3.0-service \
-- 
2.45.2

